packages/tupaia-web/src/features/DashboardItem/ExpandItemButton.tsx

```diff
@@ -33,16 +33,15 @@ const ExpandableButton = styled(Button).attrs({
     position: absolute;
     width: 100%;
     height: 100%;
-    background-color: ${({ theme }) => theme.palette.common.black};
     border: none;
     top: 0;
     left: 0;
     opacity: 0;
     margin-top: 0;
     &:hover,
     &:focus-visible {
-      opacity: 0.7;
-      background-color: ${({ theme }) => theme.palette.common.black};
+      background-color: rgba(32, 33, 36, 0.6);
+      opacity: 1;
```

Comment:
If you have an `rgba` value then you don't need `opacity` set as well, since the `a` in `rgba` is the opacity of the colour